ONSdigital/dp-dd-react-app
# Initial Jenkinsfile

### What

Added initial `Jenkinsfile`.

### How to review

* Check the build pipeline for this branch, should be passing
* Check the resource has been uploaded to the CDN

### Who can review

Anyone but myself.
